bin/common.sh

```diff
@@ -66,6 +66,7 @@ fi
 ZEPPELIN_CLASSPATH+=":${ZEPPELIN_CONF_DIR}"
 
 function check_java_version() {
+    [[ ${JAVA_HOME+x} ]] && JAVA="$JAVA_HOME/bin/java"
```

Comment:
Please use the syntax `if ; then fi` to be consistent with the rest.

Reply:
Changed to if statement.